app/src/main/java/com/github/se/studybuddies/utility/GroupInvitation.kt

```diff
@@ -0,0 +1,64 @@
+package com.github.se.studybuddies.utility
+
+import android.util.Log
+import com.github.se.studybuddies.database.DatabaseConnection
+
+private val db = DatabaseConnection()
+
+// Function to create a group invitation link
+suspend fun createGroupInviteLink(groupUID: String, groupName: String = ""): String {
+  return if (groupUID == "") {
+    Log.d("Link", "The Group id is empty")
+    "Group_not_founded"
+  } else {
+    var newGroupName = groupName
+    if (groupName == "") {
+      newGroupName = db.getGroupName(groupUID)
+      if (newGroupName == "") {
+        newGroupName = "NotNamedGroup"

```

Comment:
I feel like "Not a group name" would be better, also need to be extracted into strings.xml

Reply:
I think I will juste put "" to avoid any confusion